src/organisation/store/reducers/organisation.reducer.ts

```diff
@@ -125,6 +125,15 @@ export function reducer(
       }
       return state;
 
+    case fromOrganisation.ORGANISATION_UPDATE_PROFILE_IDS:
+      return {
```

Comment:
A page refresh whilst on the `/users`/ page may dispatch this event but the reducer never never gets invokes because the org does not exist in the store